htdocs/js/components/StaticDataTable.js

```diff
@@ -94,7 +96,78 @@ StaticDataTable = React.createClass({
             identifiers: this.props.RowNameMap
         });
     },
-    render: function () {
+    countFilteredRows: function countFilteredRows() {
+
+        var filterMatchCount = 0;
+        var filterValuesCount = this.props.Filter ? Object.keys(this.props.Filter).length : 0;
+        var tableData = this.props.Data;
+        var headersData = this.props.Headers;
+
+        for (var i = 0; i < tableData.length; i++) {
+
+            var headerCount = 0;
+
+            for (var j = 0; j < headersData.length; j++) {
+                var data = tableData[i] ? tableData[i][j] : null;
+                if (this.hasFilterKeyword(headersData[j], data)) {
+                    headerCount++;
+                }
+            }
+
+            if (headerCount === filterValuesCount) {
+                filterMatchCount++;
+            }
+        }
+
+        var hasFilters = filterValuesCount !== 0;
+        if (filterMatchCount === 0 && hasFilters) {
+            return 0;
+        }
+
+        return filterMatchCount === 0 ? tableData.length : filterMatchCount;
+    },
+    toCamelCase: function toCamelCase(str) {
+        return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function (match, index) {
+            if (+match === 0) return "";
```

Comment:
I think the `+` before `match` is probably a typo. If not, you should include a comment explaining the reasoning.